src/views/authenticated/List.tsx

```diff
@@ -16,17 +19,47 @@ export function List({ data: unfilteredListItems }: Props) {
 		);
 	}, [searchTerm, unfilteredListItems]);
 
+	const handleAddItemsClick = () => {
+		navigate("/manage-list");
+	};
+
 	return (
 		<>
 			<p>
 				Hello from the <code>/list</code> page!
 			</p>
 
+			{unfilteredListItems.length === 0 && (
+				<div>
+					<h2>Your list is ready!</h2>
+					<h3>
+						It looks like you haven’t added any items yet.
+						<br />
+						Let’s get started by adding your first item.
+					</h3>
+					<button
+						onClick={handleAddItemsClick}
```

Comment:
A saying I _love_ is "avoid shallow abstractions" - a phrase I personally grabbed from John Ousterhout who authored the book "A Philosophy of Software Design" - this talk is a great introduction to that: https://www.youtube.com/watch?v=bmSAYlu0NcY

Try reading this from the perspective of someone who is first reading this component for the first time, or from the perspective of yourself reading this 6 months from now: can you figure out what `handleAddItemsClick` means? You'd probably need to click into the function, only to find out it's actually a wrapper around a single function call. It's actually a lot easier to communicate what this button click does by just inlining the function directly. If that function gets sufficiently complex, moving it into its own function starts to make a lot more sense